### PR TITLE
name reservation system

### PR DESCRIPTION
**Pull request by crdunwel**

### CODE REVIEW

File: world/client.cpp [hunk 1]
```diff
@@ -700,7 +700,7 @@ bool Client::HandleChecksumPacket(const EQApplicationPacket *app)
 	Checksum_Struct *cs=(Checksum_Struct *)app->pBuffer;
 	uint64 checksum = cs->checksum;
 
-	if (!RuleB(Quarm, EnableChecksumEnforcement))
+	if (!RuleB(Quarm, EnableChecksumEnforcement))
```
Comment (crdunwel) [inline world/client.cpp:703]: Stared at this far too long idk why it's being counted as a change

**Comment by SecretsOTheP** [comment]
Commit looks solid. I've left a few changes to make in the in-line comments, then I'll accept the PR.

**Comment by crdunwel** [comment]
Where are these suggested changes?

**Comment by SecretsOTheP** [comment]
This is merged in as of today with the suggestions I made above. Thank you so nuch

**Pull request closed without merging**